src/form.js

```diff
@@ -764,11 +741,13 @@ define([
             //    this.fixBrokenReferences(mug);
             //}
             if (mug.options.isODKOnly) {
-                this.updateError({
+                // is this a good candidate for "info" message level?
+                mug.addMessage(null, {
+                    key: 'form-odk-only-warning',
+                    level: 'warning',
```

Comment:
@benrudolph now that I've done this I'm wondering if it was worth it. The [constants I added](https://github.com/dimagi/Vellum/commit/e14c34c605ae4db9e26924bc5cc84011669f63db) will not protect from misspellings such as `mug.ERORR` or `mug.WRANING` due to JavaScript's liberal attribute access policy. Comparisons against the values returned by those misspellings (which return `undefined`) will not raise an error either.